site/phpstan-baseline.neon

```diff
@@ -305,6 +300,16 @@ parameters:
 			count: 1
 			path: app/controllers/superuser/SuperuserEmailController.php
 
+		-
+			message: "#^Property app\\\\entities\\\\VcsAuthToken\\:\\:\\$expiration type mapping mismatch\\: property can contain DateTime\\|null but database expects DateTimeInterface\\.$#"
+			count: 1
+			path: app/entities/VcsAuthToken.php
```

Comment:
Can we add that nullable flag here?

Reply:
The nullable flag can't be used on the `@ORM\ManyToOne` annotation.  It needs a join column to be specified, which potentially changes the behavior.  The same goes for the other ignored error.